Maple2.Server.Game/Model/Field/Buff.cs

```diff
@@ -77,6 +76,16 @@ public class Buff : IUpdatable, IByteSerializable {
         }
     }
 
+    public void Reflect() {
```

Comment:
feels very strange to have this indirection here. Why can't you just check this in `AddBuff` of `BuffManager`